apps/desktop-wallet/src/utils/walletConnect.ts

```diff
@@ -16,11 +16,10 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { NetworkSettings, networkSettingsPresets } from '@alephium/shared'
+import { NetworkPreset, NetworkSettings, networkSettingsPresets } from '@alephium/shared'
```

Comment:
NIT: Not linked to this PR, but `NetworkNamePreset` would maybe be less confusing that `NetworkPreset` (as it's a subset of `NetworkName`).

Reply:
Maybe `NetworkPresetName` since it's the name of a preset rather than a preset of a name?